curriculum/challenges/english/25-front-end-development/lecture-working-with-functions/673284e7244c0c1a649121b9.md

```diff
@@ -1,76 +1,14 @@
 ---
 id: 673284e7244c0c1a649121b9
 title: What Is Scope in Programming, and How Does Global, Local, and Block Scope Work?
-challengeType: 19
-# videoId: nVAaxZ34khk
+challengeType: 11
+videoId: saSaw-VOFuo
```

Comment:
This one is set to private.

Reply:
I know, I'm waiting for Beau to be set it unlisted

Reply:
I am able to set it to unlisted on my end. But do we know if there is a particular reason why it is private?
Like is it not ready or was it marked private by accident? Did Beau say?

Reply:
Just finished watching it and it seems fine. So my guess is that it was marked private by accident.
It is unlisted now

Reply:
I think is that it's been set private by accident